modular_zubbers/code/modules/fluff/sex_barrier.dm

```diff
@@ -0,0 +1,19 @@
+/obj/item/holosign_creator/sex
+	name = "holographic sex projector"
+	desc = "Despite the name, it does not project sex, however it does create a non-blocking barrier that informs those who wish to enter that there is sex inside."
+	max_signs = 4
+	creation_time = 10
+	holosign_type = /obj/structure/holosign/sexsign
+	//The janitor sign is also purple so no need to make custom sprites here.
+
+/obj/structure/holosign/sexsign
+	name = "sex sign"
+	desc = "The words flicker DON'T SEX OPEN INSIDE. I think this means that there is sex beyond this door and that you should probably not enter, unless of course you are prepared for unforeseen consequences.
```

Comment:
```suggestion
	desc = "The words flicker DON'T SEX OPEN INSIDE. I think this means that there is sex beyond this door and that you should probably not enter, unless of course you are prepared for unforeseen consequences.“
```

Reply:
Yeah I screwed up my suggestion.

Reply:
You put the wrong airquote symbol there. Fixed it for ya